### ISSUE TITLE
CATC23: Simulate “Unable to add Node” from CLB

### ISSUE THREAD

**Issue by kivattik** [rackerlabs/otter#1331]
Simulate getting a 422 on scale up from CLB when in PENDING_DELETE which should cause the group to error

- Create a group with non-min servers and a CLB (or more)
- Set mimic to return 422 and state PENDING_DELETE on the CLB
- Attempt to scale up
- Verify that the group goes into error state since it cannot take action

Prereq: MUST be able to use Mimic to simulate incorrect CLB behavior

**Comment by cyli** [comment]
We checked with CLB, and for bulk methods, we've confirmed that 422 seems to only happen if the nodes are not present on the CLB (https://github.com/rackerlabs/mimic/issues/213).  Which for autoscale's use case counts as a success (for removing).

On 500's, we retry.

Did we want to modify this test case to be that autoscale succeeds and proceeds happily if the user had manually already removed the servers from the CLB?  Or just skip this test case?
